istio/pkg/controllers/istioconnector/controller_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package istioconnector
+
+import (
+	"testing"
+	"time"
+
+	"github.com/apache/servicecomb-service-center/istio/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDebouncing(t *testing.T) {
+	pushed := 0
```

Comment:
please use t.Run to seperate test case, and the name should be given......when....then....

consider about read UT "FIRST" principle